### PR TITLE
surveyDate check

### PR DESCRIPTION
**Pull request by ritazagoni**
In the table `KoboBnsAnswer` for the column `surveyDate`: if the year in `today` is less than `2014`, then map `2020`. If it's `2014` or more, map as before.

### CODE REVIEW

File: bns/survey.js [hunk 1]
```diff
@@ -129,8 +129,12 @@ upsert('WCSPROGRAMS_KoboBnsAnswer', 'AnswerId', {
   SubmissionUuid: dataValue('_uuid'),
   AnswerId: dataValue('_id'),
   LastUpdate: new Date().toISOString(),
-  SurveyDate: state =>
-    state.data.today ? state.data.today : state.data._submission_time,
+  SurveyDate: state => {
+    if (state.data.today.split('-')[0] >= 2014 ) {
```
Comment (lakhassane) [inline bns/survey.js:133]: Here I believe today is a string so to compare it with `2014`you might need to do this: 
`if(Number(state.data.today.split('-')[0]) >= 2014)...`

File: bns/survey.js [hunk 2]
```diff
@@ -129,8 +129,12 @@ upsert('WCSPROGRAMS_KoboBnsAnswer', 'AnswerId', {
...
+      return state.data.today ? state.data.today : state.data._submission_time
```
Comment (lakhassane) [inline bns/survey.js:134]: if you reach this point in the code, that means `state.data.today` exists already so if you want to use either `today` or `_submission_time` I would suggest before the if you add something like: 
`const date = state.data.today || state.data._submission_time`
and on your if you will use date like `if(Number(date.split('-')[0]...)`

**Review by lakhassane — COMMENTED** [review]
@ritazagoni it looks correct. There are just 2 minor changes for what you want to return.

File: bns/survey.js (hunk 1 above)
Comment (ritazagoni) [reply bns/survey.js:133]: Ah got it, thanks!

**Pull request merged**